File: erezervisi_mobile/erezervisi_mobile/lib/models/responses/user/user_get_short_dto.dart
class UserGetShortDto {
  late num id;
  late String fullName;
  late String? image;
  String? initials;

  UserGetShortDto(
      {required this.id, required this.fullName, this.image, this.initials});

  factory UserGetShortDto.fromJson(Map<String, dynamic> json) {
    return UserGetShortDto(
        id: json['id'] as num,
        fullName: json['fullName'] as String,
        image: json['image'],
        initials: json['initials']);
  }
}

File: erezervisi_api/eRezervisi/eRezervisi.Common.Dtos/User/UserGetShortDto.cs
namespace eRezervisi.Common.Dtos.User
{
    public class UserGetShortDto
    {
        public long Id { get; set; }
        public string FullName { get; set; } = null!;
        public string? Image { get; set; }
        public string Initials { get; set; } = null!;
    }
}
